samigo/samigo-api/src/java/org/sakaiproject/tool/assessment/data/ifc/assessment/EvaluationModelIfc.java

```diff
@@ -33,20 +33,22 @@
     extends java.io.Serializable
 {
 
-  public static final Integer ANONYMOUS_GRADING = Integer.valueOf(1);
-  public static final Integer NON_ANONYMOUS_GRADING = Integer.valueOf(2);
-  public static final Integer GRADEBOOK_NOT_AVAILABLE = Integer.valueOf(0);
-  public static final Integer TO_DEFAULT_GRADEBOOK = Integer.valueOf(1);
+  public static final Integer ANONYMOUS_GRADING = 1;
+  public static final Integer NON_ANONYMOUS_GRADING = 2;
+  public static final Integer GRADEBOOK_NOT_AVAILABLE = 0;
+  public static final Integer TO_DEFAULT_GRADEBOOK = 1;
+  public static final String TO_DEFAULT_GRADEBOOK_STR = "1";
```

Comment:
This PR looks good. My only comment: I'd prefer you call String.valueOf(TO_DEFAULT_GRADEBOOK_STR) or Integer.toString instead of using a separate constant to record the string value

Reply:
Yep, good call. I'll update.